src/mcts/search.cc

```diff
@@ -77,6 +79,8 @@ void Search::PopulateUciParams(OptionsParser* options) {
   options->Add<FloatOption>(kCpuctStr, 0.0f, 100.0f, "cpuct") = 1.2f;
   options->Add<FloatOption>(kTemperatureStr, 0.0f, 100.0f, "temperature") =
       0.0f;
+  options->Add<IntOption>(kMinimumTemperatureVisitsStr, 0, 1000,
```

Comment:
The lower bound probably should be one here.

Reply:
Isn't 0 for disabling this feature?

Reply:
I made some changes, based on @oscardssmith and @sergiovieri's comments. Zero seems like what off should be.

Reply:
I just realised option is minimum temperature visits, not visits to ignore in temperature, so currently description is not consistent with code behaviour